src/wallet/wallet2.h

```diff
@@ -608,7 +608,7 @@ namespace tools
       bool standard = false;
       bool onetime = false;
     };
-    void check_acc_out_precomp_onetime(const crypto::public_key &spend_public_key, const cryptonote::tx_out &o, const crypto::key_derivation &derivation, const crypto::secret_key* onetime_h, size_t i, is_received_info &received, uint64_t &money_transfered, bool &error) const;
+    void check_acc_out_precomp_onetime(const crypto::public_key &spend_public_key, const cryptonote::tx_out &o, const std::pair<crypto::secret_key, crypto::public_key>& derivation_info, const crypto::secret_key* onetime_c, size_t i, is_received_info &received, uint64_t &money_transfered, bool &error) const;
```

Comment:
This breaks the refresh optimization (the reason it's called "precomp"). I would prefer that we didn't regress performance there, even if it's somewhat less of a concern with RingCT (only 2 outputs in most cases).